### PR TITLE
Feature/lga 1865 login chs newly created user

### PR DESCRIPTION
**Pull request by psweeting1**
- Refactored step where scenario outline was used for creating a new test operator. 
- Updated step I am taken to the list of operators page to work with new dictionaries created for login and creating new user.
- Add step in feature file to login into "FOX_ADMIN_NEW_USER"

### CODE REVIEW

File: behave/features/steps/foxadmin.py [hunk 1]
```diff
@@ -134,15 +135,14 @@ def step_impl(context, action):
...
 def step_impl(context):
     # Find the question by details
     # Find corresponding input and insert value from provide
-    for row in context.table:
-        text = row['details']
-        value = row['provide']
-        xpath = f".//label[text()='{text}']/following-sibling::input"
-        context.helperfunc.find_by_xpath(xpath).send_keys(value)
+    new_user = FOX_ADMIN_NEW_OPERATOR
+    for label in new_user.values():
```
Comment (jogilder) [inline behave/features/steps/foxadmin.py:143]: I would be tempted to call this element or item something other than label as is it slightly confusing as you have labels and other things in there.

**Review by jogilder — DISMISSED** [review]
Line 143 of foxadmin.py. I would be tempted to call this element or item something other than label as is it slightly confusing as you have labels and other things in there. But otherwise looks good and feel free to merge once name changed and commits squashed.

File: behave/features/constants.py [hunk 2]
```diff
@@ -33,9 +33,16 @@
...
+    "FOX_ADMIN_NEW_USER": {"username": "elvis.presley",
+                           "password": "rockandroll",
+                           "login_url": f"{CLA_BACKEND_URL}/admin/login",
+                           "user_type": "OPERATOR",
+                           "application": "BACKEND"},
 }
-
-FOX_ADMIN_NEW_OPERATOR = "elvis.presley"
+# value_key is linked to FOX_ADMIN_NEW_USER
+FOX_ADMIN_NEW_OPERATOR = {"username": {"label": "Username:", "value_key": "username"},
```
Comment (said-moj) [inline behave/features/constants.py:43]: Is this used to describe the form fields when creating a new operator? If so what do you think of adding FORM_FIELDS or something similar to constant name?

**Review by Pheonnexx — APPROVED** [review]
LGTM 👍

File: behave/features/constants.py (hunk 2 above)
Comment (psweeting1) [reply behave/features/constants.py:43]: Cool, I like it. I've added `FOX_ADMIN_FORM_FIELDS`, sound good?

**Pull request merged**